document-generator-prosecution/src/main/java/uk/gov/companieshouse/document/generator/prosecution/mapping/model/prosecutioncase/ProsecutionCase.java

```diff
@@ -1,54 +1,20 @@
 package uk.gov.companieshouse.document.generator.prosecution.mapping.model.prosecutioncase;
 
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
-
-import java.time.LocalDateTime;
-import java.util.Map;
+import uk.gov.companieshouse.api.model.prosecution.prosecutioncase.ProsecutionCaseStatusApi;
 
 public class ProsecutionCase {
-    @JsonProperty("etag")
-    private String etag;
-
-    @JsonProperty("kind")
-    private String kind;
-
-    @JsonProperty("status")
-    private ProsecutionCaseStatus status;
 
-    @JsonProperty("company_incorporation_number")
+    @JsonProperty("CompanyIncorporationNumber")
     private String companyIncorporationNumber;
 
-    @JsonProperty("company_name")
+    @JsonProperty("CompanyName")
     private String companyName;
 
-    @JsonProperty("compliance_case_id")
-    private String complianceCaseId;
-
-    @JsonProperty("compliance_user_id")
-    private String complianceUserId;
-
-    @JsonProperty("submitted_at")
-    private LocalDateTime submittedAt;
-
-    @JsonProperty("links")
-    private Map<String, String> links;
-
-    public String getKind() {
-        return kind;
-    }
-
-    public void setKind(String kind) {
-        this.kind = kind;
-    }
-
-    public ProsecutionCaseStatus getStatus() {
-        return status;
-    }
-
-    public void setStatus(ProsecutionCaseStatus status) {
-        this.status = status;
-    }
+    @JsonIgnoreProperties("status")
+    private ProsecutionCaseStatusApi status;
```

Comment:
If properties are being ignored, can they be removed from the model?

Reply:
Refactored changes for this in the DocInfoService, the Handler and the Service.